### ISSUE TITLE
Uncertain about FPN settings

### ISSUE THREAD

**Issue by wondervictor** [open-mmlab/mmdetection#298]
In `faster_rcnn_r101_fpn_1x.py`, FPN setting is:
```
    neck=dict(
        type='FPN',
        in_channels=[256, 512, 1024, 2048],
        out_channels=256,
        num_outs=5),
```
I feel doubt about whether the model only uses P2,P3,P4,P5 without P6, which is not consistent with the standarad FPN.
If we set `add_extra_convs=True` explicitly, the FPN model in mmdetction will have 5 feature pyramids?

**Comment by hellock** [comment]
The setting is consistent with the standard FPN which uses a stride-2 pooling to generate P6. `add_extra_convs=True` is used in RetinaNet.

**Comment by wondervictor** [comment]
Thanks a lot. The mAP of Faster-r101-FPN in mmdetection is 38.6 while Detectron or maskrcnn-benchmark can reach 39.4 and 39.1 respectively.  I wonder why this large gap exists.

**Comment by hellock** [comment]
The backbone is different. Detectron and maskrcnn-benchmark use caffe-style backbones and mmdetection uses pytorch-style backbones by default (in order to use pretrained models in pytorch model zoo).
You can find the performance of both in [model zoo](https://github.com/open-mmlab/mmdetection/blob/master/MODEL_ZOO.md#faster-r-cnn). The caffe-style backbone gets 38.8 mAP, which is still a little lower than Detectron. This may be an exception since other models (R-50 Faster R-CNN, R-50/101 Mask R-CNN) achieve similar performance.

**Comment by wondervictor** [comment]
okay, I am clear now. Thanks
